cypress/component/DataSearch/dataset_search_table.spec.js

```diff
@@ -38,6 +42,56 @@ describe('Dataset Search Table tests', () => {
       cy.get('#header-checkbox').click();
       cy.contains('1 dataset selected from 1 study');
     });
+  });
+
+
+  describe('Data library filter by participant count tests', () => {
+
+    beforeEach(() => {
+      cy.initApplicationConfig();
+      cy.stub(TerraDataRepo, 'listSnapshotsByDatasetIds').returns({});
+      cy.clock();
+    });
+
+    function handler(request, searchText) {
+      if (JSON.stringify(request.body).includes(searchText)) {
+        request.reply(['filtered']);
+      } else {
+        request.reply([]);
+      }
+    }
+
+
+    it('When a participant count filter is applied the query is updated', () => {
+      cy.intercept(
+        {method: 'POST', url: '**/search/index'}, (req) => {
+          return handler(req, '{"range":{"participantCount":{"gte":null,"lte":50}}}');
+        }).as('searchIndex');
+      mount(<DatasetSearchTable {...props} />);
+      // first clear the default value (100), without clearing first, type('50') would result in input of 10050
+      cy.get('#participantCountMax-range-input').clear().type('50');
+      cy.tick(150);
```

Comment:
Cypress recommends that there be a `clock` command that precedes the `tick` command: https://docs.cypress.io/api/commands/tick

Reply:
I use the clock command in the before each method, but I could move it into each individual test

Reply:
Thank you, I did miss that. No need if it's there, that should cover it.